pkg/cassandra/config.go

```diff
@@ -14,7 +15,7 @@ import (
 
 const (
 	SystemReplicationDcNames = "-Dcassandra.system_distributed_replication_dc_names"
-	SystemReplicationFactor  = "-Dcassandra.system_distributed_replication_per_dc"
+	SystemReplicationFactor  = "-Dcassandra.system_distributed_replication"
```

Comment:
The `SystemReplicationFactor` name worked nicely before since it was fed with a single number. I'd go for `SystemReplicationStrategy` or just `SystemReplication`  maybe.